packages/zk-transfer-web/src/styles/app.css

```diff
@@ -140,6 +141,12 @@ input:disabled {
     flex: 0 0 26px;
 }
 
+.token button {
```

Comment:
where is the `.token button` class used? I couldn't find a `button` anywhere in a `<div class="token">`

Reply:
That's the button i added the class names (`focus:bg-gray-200 focus:outline-none`) to.